src/ScaleUnitManagementTests/WorkloadDataPipelineTest.cs

```diff
@@ -0,0 +1,103 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using FluentAssertions;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CloudAndEdgeLibs.Contracts;
+using ScaleUnitManagement.WorkloadSetupOrchestrator;
+using ScaleUnitManagement.Utilities;
+using Moq;
+using ScaleUnitManagement.WorkloadSetupOrchestrator.Utilities;
+using CloudAndEdgeLibs.AOS;
+
+namespace ScaleUnitManagementTests
+{
+
+    [TestClass]
+    public sealed class WorkloadDataaPipelineTest
```

Comment:
[nit] only because it makes it harder to find using code search on GitHub 😊
```suggestion
    public sealed class WorkloadDataPipelineTest
```